src/Tizen.NUI/src/public/WindowSystem/QuickPanelService.cs

```diff
@@ -0,0 +1,174 @@
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public enum QuickPanelType
+        {
+            /// <summary>
+            /// Unknown
+            /// </summary>
+            Unknown = 0x0,
+            /// <summary>
+            /// System default
+            /// </summary>
+            SystemDefault = 0x1,
+            /// <summary>
+            /// Context menu
+            /// </summary>
+            ContextMenu = 0x2,
+        }
+
+        /// <summary>
+        /// Creates a new Quickpanel Service handle.
+        /// </summary>
+        /// <param name="tzShell">The TzShell instance.</param>
+        /// <param name="win">The window to provide service of the quickpanel.</param>
+        /// <param name="type">The type of quickpanel service.</param>
+        /// <exception cref="Tizen.Applications.Exceptions.OutOfMemoryException">Thrown when the memory is not enough to allocate.</exception>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="ArgumentNullException">Thrown when a argument is null.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public QuickPanelService(TizenShell tzShell, Window win, QuickPanelType type)
+        {
+            if (tzShell == null)
+            {
+                throw new ArgumentNullException((string)"tzShell");
+            }
+            if (tzShell.GetNativeHandle() == (IntPtr)0)
+            {
+                throw new ArgumentException("tzShell is not initialized.");
+            }
+            if (win == null)
+            {
+                throw new ArgumentNullException((string)"win");
+            }
+
+            _tzsh = tzShell;
+            _tzshWin = win.GetNativeId();
+            _tzshQpService = Interop.QuickPanelService.tzshQuickPanelServiceCreateWithType(_tzsh.GetNativeHandle(), (IntPtr)_tzshWin, (int)type);
+            if (_tzshQpService == (IntPtr)0)
+            {
+                int err = Tizen.Internals.Errors.ErrorFacts.GetLastResult();
+                _tzsh.ErrorCodeThrow(err);
+            }
+        }
+
+        /// <summary>
+        /// Destructor.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        ~QuickPanelService()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Dispose.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <inheritdoc/>
+        protected virtual void Dispose(bool disposing)
+        {
+            int _res;
+            if (!disposed)
+            {
+                if (_tzshQpService != (IntPtr)0)
```

Comment:
Use IntPtr.Zero instead of (IntPtr)0 in all your code.

Reply:
I'll change all of (IntPtr)0